pkg/resources/server.go

```diff
@@ -115,8 +115,8 @@ func (rs *resourceServer) NotifyRegistrationStatus(ctx context.Context,
 	if regstat.PluginRegistered {
 		glog.Infof("Plugin: %s gets registered successfully at Kubelet\n", rs.endPoint)
 	} else {
-		glog.Infof("Plugin: %s failed to be registered at Kubelet: %v; restarting.\n", rs.endPoint, regstat.Error)
 		rs.grpcServer.Stop()
+		glog.Fatalf("Plugin: %s failed to be registered at Kubelet: %v; restarting.\n", rs.endPoint, regstat.Error)
```

Comment:
im not sure im OK with logging a msg then panic,

maybe we need some way to signal errors and exit gracefully ?
what others think of this ?

Reply:
sure we can switch to log.err and exit 1 or something like that

Reply:
isn't log.Fatal the same as log and exit 1?